content/post/twitter-hashtags-real-time.md

```diff
@@ -0,0 +1,561 @@
+
+Hanee' is a Big Data Engineer, with experience working with massive data in many industries, such as Telecommunications and Banking.
+
+## Overview
+
+One of the primary places where the world is seeing an explosion of data growth is in social media.  [Wallaroo](http://www.wallaroolabs.com/community) is a powerful and simple-to-use open-source data engine that is ideally suited for handling massive amounts of streaming data in real-time.
+
+In this tutorial, I will use Wallaroo to analyze and extract insights from Twitter in real-time and present the results on a dashboard.
+
+First, a little background on Wallaroo.  Wallaroo is a relatively new open-source project that has been getting a lot of attention recently.  Wallaroo Labs, the company behind the project, has shared a good deal of information about their approach and technology. You may have seen some of their recent blog articles on Hacker News.
+
+You can find more information about Wallaroo by visiting [this site](wallaroolabs.com/community).
+
+Wallaroo allows developers to write code in native Python and, unlike other streaming projects, doesn't require using Java or the JVM. I was intrigued by their approach and wanted to see how easy it would be to use Wallaroo to do some analysis on Twitter data.
+
+## Tutorial
+
+This post shows a real use case on a massive online data stream, using Wallaroo’s Python API. We will show how easy it is to transform data streams with a small amount of code.
+
+We will create an application that reads a real data stream from Twitter, extracts hashtags, and counts them to identify the top trending hashtags on Twitter. You can create the needed files on your own or follow along by cloning the [Wallaroo Twitter Trending Example](https://github.com/WallarooLabs/wallaroo-twitter-trending-example) from GitHub.
+
+
+### 1. Install Wallaroo
+
+Before we get started, you should make sure you have Wallaroo installed.  You can find detailed instructions [here](https://docs.wallaroolabs.com/book/getting-started/setup.html).
+
+
+
+### 2. Register for Twitter APIs
+
+In order to get real-time tweets, you need to register on [Twitter
+Apps](https://apps.twitter.com/) by clicking on “Create new app”, and
+filling in the form under “Create your twitter app”.
+
+![](/images/post/twitter-hashtags-real-time/credentials.png)
+
+Go to your newly created app and open the “Keys and Access
+Tokens” tab, then click on “Create my access token”.
+
+![](/images/post/twitter-hashtags-real-time/credentials_2.png)
+
+Your new access tokens will appear like this:
+
+![](/images/post/twitter-hashtags-real-time/credentials_3.png)
+
+
+
+### 3. Create Twitter Client
+
+Let's start by creating a client that connects to the Twitter API in order to grab the tweets and send them to Wallaroo.
+
+**twitter_client.py** connects to Wallaroo using a TCP connection, calls the Twitter Streaming API to get the tweets in real-time, and forwards them to Wallaroo in order to be processed.
```

Comment:
`**...**` bold doesn't seem to render on our blog. You should use code tags instead

Reply:
Fixed.